lib/core/src/Cardano/Wallet/Network.hs

```diff
@@ -49,6 +52,10 @@ data NetworkLayer t m = NetworkLayer
     , postTx
         :: (Tx t, [TxWitness]) -> ExceptT ErrPostTx m ()
         -- ^ Broadcast a transaction to the chain producer
+
+    , decodeExternalTx
+        :: ByteString -> ExceptT ErrDecodeExternalTx m (Tx t, [TxWitness])
+        -- ^ Decode an externally signed transaction to the chain producer
```

Comment:
Just a nitpick: I would propose using hyphenation in compound adjectives such as this one. So for example: "externally-signed transaction" rather than "externally signed transaction".

Reply:
done